index.css

```diff
@@ -0,0 +1,540 @@
+@import url('http://fonts.cdnfonts.com/css/cocogoose');
+
+* {
+  margin: 0;
+  padding: 0;
+  outline: 0;
+  list-style: none;
+  text-decoration: none;
+  box-sizing: border-box;
+}
+
+header {
+  position: absolute;
+  width: 100%;
+  background: #fff;
+  box-shadow: 4px 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+nav .bottom {
+  width: 85%;
+  margin: 0 auto;
+  padding: 0.5rem;
+  display: flex;
+  align-items: center;
+  gap: 30%;
+}
+
+nav .bottom .logo {
+  order: 2;
+  font-family: 'Lato', sans-serif;
+  font-weight: 800;
+  font-size: 1.5rem;
+  color: #515151;
+}
+
+nav .top {
+  display: none;
+}
+
+.nav-menu,
+.nav-btn {
+  display: none;
+}
+
+.hamburger {
+  display: block;
+  margin-left: 0;
+  z-index: 999;
+}
+
+@media only screen and (max-width: 768px) {
+  .hamburger.active .bar:nth-child(2) {
+    opacity: 0;
+  }
+
+  .hamburger.active .bar:nth-child(1) {
+    transform: rotate(45deg) translateY(12px);
+  }
+
+  .hamburger.active .bar:nth-child(3) {
+    transform: rotate(-45deg) translateY(-11px);
+  }
+
+  .nav-menu {
+    position: fixed;
+    right: -100%;
+    height: 100vh;
+    z-index: 100;
+    top: -2px;
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+    padding: 5rem 0;
+    align-items: center;
+    background-color: #ff5438;
+    mix-blend-mode: multiply;
+    backdrop-filter: blur(6px);
+    width: 100%;
+    transition: 0.5s;
+  }
+
+  .nav-menu.active {
+    right: 0;
+    z-index: 50;
+    overflow: hidden;
+  }
+
+  .nav-link {
+    font-family: 'Lato', sans-serif;
+    font-weight: 800;
+    font-size: 1.5rem;
+    color: #fff;
+    z-index: 9999;
+  }
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 5px;
+  border-radius: 1rem;
+  margin: 3px auto;
+  background-color: #595959;
+}
+
+.hero {
+  width: 100%;
+  background-image: url('./images/mobile-bg.png');
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.hero-wrapper {
+  width: 85%;
+  margin: 0 auto;
+  padding: 5rem 1rem 5rem 0;
+}
+
+.hello {
+  font-family: 'Lato', sans-serif;
+  color: #ff5438;
+  font-weight: 400;
+  font-size: 1rem;
+  line-height: 1rem;
+}
```

Comment:
- [OPTIONAL] I think it would be better to implement the following for **_"Hello young entrepreneurs"_** for desktop version 👇 

```
  font-size: 1.7rem;
  letter-spacing: 0.5px;
  padding-bottom: 0.5rem;
```